### PR TITLE
Reviews: 

### PR DESCRIPTION
**Pull request by raphaelspies**
- Added Dropdown menu with SortBy function (changes sort order of reviews)
- Moved shared styles to styles.jsx

### CODE REVIEW

File: client/src/components/Reviews/Dropdown.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+//     this.state = {
+//       isListOpen: false,
+//       sortBy: this.props.sortBy,
+//       options: [
+//         "date",
+//         "Helpfulness"
+//       ]
+//     }
+//   }
+// }
```
Comment (Lordnickim) [inline client/src/components/Reviews/Dropdown.jsx]: Still need the reference?

File: client/src/components/Reviews/ReviewList.jsx [hunk 2]
```diff
@@ -1,33 +1,44 @@
...
+        return -1;
+      } else if (a[key] < b[key]) {
+        return 1;
+      }
+    }
+  }
+
+  function changeSortOrder(e){
+    setSortBy(e.target.value)
+    console.log(e.target.value)
```
Comment (Lordnickim) [inline client/src/components/Reviews/ReviewList.jsx]: Still need ref here too?

**Review by Lordnickim — COMMENTED** [review]
Maybe another commit without the console and commented out code?

File: client/src/components/Reviews/ReviewList.jsx (hunk 2 above)
Comment (raphaelspies) [reply client/src/components/Reviews/ReviewList.jsx]: > Maybe another commit without the console and commented out code?

Will re-commit with changes

**Review by erikeh — APPROVED** [review]

**Review by Lordnickim — APPROVED** [review]

**Pull request merged**